connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -132,6 +136,27 @@ public String version() {
         return AppInfoParser.getVersion();
     }
 
+    @Override
+    public boolean alterOffsets(Map<String, String> connectorConfig, Map<Map<String, ?>, Map<String, ?>> offsets) {
```

Comment:
What's the benefit of allowing users to modify the offsets here? It seems to me like it's essentially a no-op, so would it not be more appropriate to simply throw an `UnsupportedOperationException` with a relevant message?